languages/csharp/exercises/concept/overflow/.docs/after.md

```diff
@@ -0,0 +1,48 @@
+The exercise shows the behavior of various numeric types when they overflow, i.e. when their capacity is insufficient to contain the value resulting from a computation such as an arithmetic operation or cast.
+
+- unsigned [integers][integral-numeric-types] (`byte`, `ushort`, `uint`, `ulong`) will wrap around to zero (the type's maximum value + 1 acts as a modulus) unless [broadly speaking][checked-compiler-setting] they appear within a [`checked`][checked-and-unchecked] block in which case an instance of `OverflowException` is thrown. `int` and `long` will behave similarly except that they wrap around to `int.MinValue` and `long.minValue` respectively.
```

Comment:
When testing things, I just found at that the C# compiler can detect overflows when using constants:

```csharp
var expr = int.MaxValue + 1;
```

This code leads to a compiler error:

>  Program.cs(19, 28): [CS0220] The operation overflows at compile time in checked mode

Might be nice to mention this somewhere too.

Reply:
Done